torch/nn/functional.py

```diff
@@ -3639,9 +3639,11 @@ def multi_head_attention_forward(query,                           # type: Tensor
         - value: :math:`(S, N, E)` where S is the source sequence length, N is the batch size, E is
           the embedding dimension.
         - key_padding_mask: :math:`(N, S)`, ByteTensor, where N is the batch size, S is the source sequence length.
+          ``1`` or ``True`` is filled with ``-inf`` while ``0`` or ``False`` is filled with ``0.0``.
         - attn_mask: 2D mask :math:`(L, S)` where L is the target sequence length, S is the source sequence length.
           3D mask :math:`(N*num_heads, L, S)` where N is the batch size, L is the target sequence length,
-          S is the source sequence length.
+          S is the source sequence length. If a ByteTensor is provided, it will be converted to a float tensor.
```

Comment:
I don't think you need to mention that it'll be converted, but you should mention how its semantics differ.